docs/maintaining.md

```diff
@@ -160,22 +190,39 @@ curl -X PUT --header 'Content-Type: application/json' -d '
 
 ### Resetting your GraphDB instance
 
-If you previously set up a Neurobagel node on your machine but want to reset your graph database to start again _from scratch_, 
-the most foolproof way would be to start with a clean GraphDB configuration to avoid conflicts with any previously created credentials or databases.
+Each Neurobagel node has its own GraphDB instance, which is used to store the graph data for the node.
+If you want to reset your graph database and start again from scratch,
+follow these steps:
+
+1. Ensure that your Neurobagel node is not running (i.e., shut down the Docker containers for the node).
```

Comment:
In the docs preview I don't see these numbered points rendered correctly - could you double check this?

Reply:
Interesting, that seems to be true of all the numbered lists here. Not sure why

Reply:
seems related, but apparently this was fixed a long time ago: https://github.com/squidfunk/mkdocs-material/discussions/3294

Reply:
OK, tailwind was the culprit here. In fact, our production docs don't have any visible ordered list elements right now. Kinda spooky - maybe we need some visual integration tests ...